effekt/shared/src/main/scala/effekt/typer/ExhaustivityChecker.scala

```diff
@@ -152,7 +160,8 @@ object ExhaustivityChecker {
     def reportNonExhaustive()(using C: ErrorReporter): Unit = {
       @tailrec
       def traceToCase(at: Trace, acc: String): String = at match {
-        case Trace.Root(_) => acc
+        case Trace.Root(_) if originalScrutinees.length == 1 => acc
+        case Trace.Root(e) => originalScrutinees.map { f => if e == f then acc else "_" }.mkString(", ")
```

Comment:
Could you also add a neg test testing this new message?

Reply:
Done.